packages/lit-dev-content/src/components/litdev-example-controls.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {LitElement, html, css, customElement, property} from 'lit-element';
+import './litdev-typescript-switch.js';
+
+/**
+ * Controls for lit.dev code examples.
+ */
+@customElement('litdev-example-controls')
+export class LitDevExampleControls extends LitElement {
```

Comment:
Minor style nitpick. The switch and openInPlayground link have different heights, 25px and 22px respectively.

This is non blocking since the switch isn't even enabled. Instead commenting that it currently feels like the switch is sitting lower in the menu bar than the open in playground image. The two menu items don't feel vertically aligned.

Reply:
Here are some options:

Before:
<img src="https://user-images.githubusercontent.com/48894/132425290-cef94bf7-4752-4642-9a1e-70328d34efdd.png" width="150px">

Top-aligned:
<img src="https://user-images.githubusercontent.com/48894/132425285-2252644a-2e1d-4da0-9ea7-f58b358138a1.png" width="150px">

Enlarged:
<img src="https://user-images.githubusercontent.com/48894/132424872-8fcb7582-6147-4eff-ae63-a22e3dae3a53.png" width="150px">

Is "top-aligned" what you were thinking? Or more like "enlarged"? The problem with "enlarged" to my eye is that the stroke width of the icon becomes too heavy, and that clashes with the stroke width of the JS/TS switch. We could edit the SVG to drop the stroke width, though. WDYT?

Reply:
Let's keep it as before. I think my eye is getting impacted by the arrow at the top of the icon making it feel like it's higher up.

I was thinking more along the lines of enlarged but 100% agree about the stroke width making it awkward.

Reply:
I like "before" of these... my understanding of the material sizing system (likely imperfect) is that there's an outer size, like 24px, but the icon doesn't fully fill it, which some components, like this switch might. So the container's hsould have the same size, but the icon-based widgets will appear a bit smaller.

Reply:
Done!